File: 502_data mining/module3/data_partitioning_example.r
#partition data in R 
set.seed(7) #this is arbitrary, but we need a random number gen

#dataset 
bank = read.csv(".csv")

#get number of rows 
n <- dim(bank)[1]

#determine which records go in our training set 
#here, about 75% will go into our training set 
train_ind <- runif(n) < 0.75 

bank_train <- bank[train_ind,]
bank_test  <- bank[!train_ind,]

File: 502_data mining/module3/data_partitioning_example.py
#HOW TO PARTITION DATA - Python
import pandas as pd
from sklearn.model_selection import train_test_split
import random

bank = pd.read_csv("#.csv")
bank_train, bank_test = train_test_split(bank, test_size =
0.25, random_state = 7)

#verify 
bank.shape
bank_train.shape
bank_test.shape